plugins/processors/ifname/ifname_test.go

```diff
@@ -0,0 +1,106 @@
+//+build localsnmp
```

Comment:
This is a switch to avoid running unless you have a local snmp server?  The normal way we would do this is to add to each test a few lines like:
```
	if testing.Short() {
		t.Skip("Skipping integration test in short mode")
	}
```

Reply:
It's a switch to turn off these tests in CI because they don't have a local snmp agent.  I've been running 'go test -tags localsnmp'.

I've changed the tests to avoid doing this.  TestIfName now mocks the snmp response.  TestTable and TestXTable are now skipped for short tests.

Reply:
Automated unit testing just reminded me that this needs snmptranslate and mibs to be installed locally so mocking the snmp response isn't enough to get TestIfName as a short test.  It's now skipped too.